cartography/intel/aws/iam.py

```diff
@@ -83,7 +82,7 @@ def get_user_policy_data(boto3_session, user_list):
         resource_user = resource_client.User(name)
         try:
             policies[arn] = {p.name: p.policy_document["Statement"] for p in resource_user.policies.all()}
-        except client.exceptions.NoSuchEntityException:
+        except resource_client.meta.client.exceptions.NoSuchEntityException:
```

Comment:
Turns out you can access the exceptions from resource by using .meta. 

Removed unnecessary instantiation of client. Much nicer this way :) Good comment before 👍